open_spiel/scripts/install.sh

```diff
@@ -150,6 +150,23 @@ if [[ ${BUILD_WITH_LIBTORCH:-"ON"} == "ON" ]] && [[ ! -d ${DIR} ]]; then
   unzip "${DOWNLOAD_FILE}" -d "open_spiel/libtorch/"
 fi
 
+# Add OrTools
+# This downloads the precompiled binaries available from the official website.
+# https://developers.google.com/optimization/install/cpp/
+DIR="open_spiel/ortools"
+if [[ ${BUILD_WITH_ORTOOLS:-"ON"} == "ON" ]] && [[ ! -d ${DIR} ]]; then
+  # You may want to replace this URL according to your system.
+  #
+  # Use version 8 at minimum, due to compatibility between absl library versions
+  # used in OpenSpiel and in OrTools.
+  DOWNLOAD_URL="https://github.com/google/or-tools/releases/download/v8.0/or-tools_ubuntu-18.04_v8.0.8283.tar.gz"
```

Comment:
Can we have an if Mac OS download X, else download Y here?

Reply:
Oops, I realized I had never sent that last message, sorry...! 

Easy to do on my side... but easier for you to test directly on Travis by just doing more commits, so let's do that. 

Basically, I think all it takes is checking that BUILD_WITH_OR_TOOLS is something other than "OFF" and copy that to DOWNLOAD_URL. And then in travis.yml just use the Ubuntu 18.04 link. 

We should clarify this in the documentation (like the main CMakeLists too, and maybe include all the common links: MacOS, Ubuntu 18.04 and Ubuntu 20.04 so people can copy/paste.

Thanks for doing this.. sorry for the run-around!